recipes/srt/all/conanfile.py

```diff
@@ -35,8 +39,8 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("openssl/1.1.1g")
-        if self.settings.os == "Windows":
+        self.requires("openssl/1.1.1h")
+        if self.settings.os == "Windows" and not self._has_stdcxx_sync:
             self.requires("pthreads4w/3.0.0")
```

Comment:
I guess that MinGW with pthread doesn't need `pthreads4w`.

Reply:
I am sorry, ive never used nor do we need mingw 
for production so I never think I about it. 🙊 

I would like this available for my team so I would prefer to tackle  that in the near future so I can open an issue for adding support... would be a good excuse for me to learn